src/services/textChanges.ts

```diff
@@ -347,6 +353,23 @@ namespace ts.textChanges {
             this.replaceRangeWithText(sourceFile, createRange(pos), text);
         }
 
+        public tryInsertJSDocParameters(sourceFile: SourceFile, parameters: codefix.ParameterInference[]) {
+            if (parameters.length === 0) {
+                return;
+            }
+            const parent = parameters[0].declaration.parent;
+            const indent = getLineAndCharacterOfPosition(sourceFile, parent.getStart()).character;
+            let commentText = "\n";
+            for (const { declaration, typeNode, isOptional } of parameters) {
+                if (isIdentifier(declaration.name)) {
+                    const printed = createPrinter().printNode(EmitHint.Unspecified, typeNode!, sourceFile);

```

Comment:
`changesToText.getNonformattedText(typeNode!, sourceFile, this.newLineCharacter).text;` works too.